# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1302613336 -7200
#      Tue Apr 12 15:02:16 2011 +0200
# Node ID d4e0d8f925111af057f1f0db60dd050317fca91d
# Parent  d47362945576d32c7e90e8fa4ff1e111a6baab64
Remove connect with slot that does not exist anymore.

diff --git a/gui//src/CodeEdit.cpp b/gui//src/CodeEdit.cpp
--- a/gui//src/CodeEdit.cpp
+++ b/gui//src/CodeEdit.cpp
@@ -73,18 +73,16 @@ CodeEdit::CodeEdit(QWidget *parent)
 
     connect(&octaveCommandTimer, SIGNAL(timeout ()), this, SLOT(octaveCommandCompletion()));
 
     completionTimer.setSingleShot(true);
     completionTimer.setInterval(200);
 
     connect(&completionTimer, SIGNAL(timeout ()), this, SLOT(buildAutoCompletionList()));
 
-    connect(this, SIGNAL( cursorPositionChanged() ), this, SLOT( cursorChanged_cb() ) );
-
     auto_indent=true;
     setAcceptDrops(false);
     if(autocompletion_ok)
     {
         completion.setWidget(this);
         completion_model=new QStringListModel(&completion);
         completion.setModel(completion_model);
         completion.setCompletionMode(QCompleter::UnfilteredPopupCompletion);
